packages/machine-extractor/src/sky/skyConfigNode.ts

```diff
@@ -18,11 +22,42 @@ export type SkyConfigNodeReturn = WithValueNodes<{
 }> &
   Pick<ObjectPropertyInfo, 'node'>;
 
+const StringLiteralOrExpression = unionType([
```

Comment:
```suggestion
const StringLiteralOrProcessEnvKey = unionType([
```

Reply:
I updated the name to `StringLiteralOrEnvKey`. I think the `Process` part was too specific as this also works for `import.meta.SOME_KEY`